instructors.md

```diff
@@ -229,3 +229,77 @@ And please also make use of our
 [Software Carpentry Windows Installer](http://github.com/swcarpentry/windows-installer).
 
 [shebang]: http://www.in-ulm.de/~mascheck/various/shebang/
+
+## Solutions to exercises
+
+## [Navigating Files and Directories](01-filedir.html)
+
+> ## Many ways to do the same thing - absolute vs relative paths {.challenge}
+>
+> Starting from a filesystem location of `/Users/amanda/data/`, 
+> which of the following commands could Amanda use to navigate to her home directory, 
+> which is `/Users/amanda`?
+> 
+>1.  `cd .` **no** (`.` stands for the current directory)
+>2.  `cd /` **no** (`/` stands for the root directory)
+>3.  `cd /home/amanda` **no** (Amanda's home directory is `/Users/amanda`)
+>4.  `cd ../..` **no** (this goes up two levels, i.e. ends in `/Users`)
+>5.  `cd ~`  **yes** (`~` stands for the user's home directory)
+>6.  `cd home` **no** (this would navigate into a directory `home` in the current directory)
+>7.  `cd ~/data/..` **yes** (complicated, but correct)
```

Comment:
How about "but correct" -> "but puts you in the right directory" to avoid qualifying the answer as "good"